### DISCUSSION TITLE
Future Ready Talent

### DISCUSSION THREAD

**Discussion by VarunSarkar** [github/india#345] [Students]
I want to ask how are we going to be updated about the mentorship sessions as I don't see any chat or notice section on frt portal nor the events section has been updated.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Reply by VarunSarkar** [reply]
man will you stop spamming every discussion. We here are have more important thinks to do than your stupid marketing and all.
